### ISSUE TITLE
Add Readme 

### ISSUE THREAD

**Issue by omaramin17** [yougar0/AssetWatchdog#1]
Hi , Thanks for your great work on this tool :)
Can you add a readme file on installation , features , etc ? @yougar0

**Comment by yougar0** [comment]
Just a moment, I'm still working on it. There are also some bugs to fix.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yougar0/AssetWatchdog"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月8日(星期四) 下午2:09
***@***.***&gt;;
抄送:&nbsp;"Charm ***@***.******@***.***&gt;;
主题:&nbsp;[yougar0/AssetWatchdog] Add Readme  (#1)





 
Hi , Thanks for your great work on this tool :)
 Can you add a readme file on installation , features , etc ? @yougar0

**Comment by omaramin17** [comment]
yeah i noticed some errors while trying to log into /admin section too 
![assetwatchdog](https://user-images.githubusercontent.com/25645659/113977935-9aaa1280-9843-11eb-9ac6-ba23aab67e8d.jpg)

Waiting for documentation bud   <3
